### ISSUE TITLE
ListMailboxes returns no mailboxes [Gmail]

### ISSUE THREAD

**Issue by MarcoCanalcar** [emailjs/emailjs-imap-client#167]
When I call listMailboxes on my gmail account, no mailboxes are returned, however they are printed in the logs. I believe the error is related to this [this](https://github.com/emailjs/emailjs-imap-client/blob/ff6f24f99d75e78559631b50c0a49f9e0292b5f1/src/client.js#L311) line. 

This is the value of 'attr' for some mailbox
```
[[{"type":"ATOM","value":"\\HasNoChildren"}],{"type":"STRING","value":"/"},{"type":"STRING","value":"somename"}]
```
which, since it's length is 3 doesn't pass the if statement

Logs

```
[DEBUG][2018-02-06T12:22:15.500Z][1][-redacted-][imap.gmail.com] Listing mailboxes...
[DEBUG][2018-02-06T12:22:15.500Z][1][-redacted-][imap.gmail.com] C: W5 LIST "" "*"
[DEBUG][2018-02-06T12:22:15.746Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename"
[DEBUG][2018-02-06T12:22:15.746Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "INBOX"
[DEBUG][2018-02-06T12:22:15.746Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename2"
[DEBUG][2018-02-06T12:22:15.746Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename3"
[DEBUG][2018-02-06T12:22:15.746Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename4"
[DEBUG][2018-02-06T12:22:15.747Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename5"
[DEBUG][2018-02-06T12:22:15.747Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasChildren \Noselect) "/" "[Gmail]"
[DEBUG][2018-02-06T12:22:15.749Z][1][-redacted-][imap.gmail.com] S: * LIST (\Drafts \HasNoChildren) "/" "[Gmail]/Borradores"
[DEBUG][2018-02-06T12:22:15.750Z][1][-redacted-][imap.gmail.com] S: * LIST (\Flagged \HasNoChildren) "/" "[Gmail]/Destacados"
[DEBUG][2018-02-06T12:22:15.750Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren \Sent) "/" "[Gmail]/Enviados"
[DEBUG][2018-02-06T12:22:15.750Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren \Important) "/" "[Gmail]/Importantes"
[DEBUG][2018-02-06T12:22:15.751Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren \Trash) "/" "[Gmail]/Papelera"
[DEBUG][2018-02-06T12:22:15.751Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren \Junk) "/" "[Gmail]/Spam"
[DEBUG][2018-02-06T12:22:15.751Z][1][-redacted-][imap.gmail.com] S: * LIST (\All \HasNoChildren) "/" "[Gmail]/Todos"
[DEBUG][2018-02-06T12:22:15.751Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename6"
[DEBUG][2018-02-06T12:22:15.752Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename7"
[DEBUG][2018-02-06T12:22:15.752Z][1][-redacted-][imap.gmail.com] S: * LIST (\HasNoChildren) "/" "somename8"
[DEBUG][2018-02-06T12:22:15.752Z][1][-redacted-][imap.gmail.com] S: W5 OK Success
[DEBUG][2018-02-06T12:22:15.752Z][1][-redacted-][imap.gmail.com] C: W6 LSUB "" "*"
[DEBUG][2018-02-06T12:22:15.844Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename1"
[DEBUG][2018-02-06T12:22:15.844Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "INBOX"
[DEBUG][2018-02-06T12:22:15.844Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename2"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename3"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename4"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename5"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasChildren \Noselect) "/" "[Gmail]"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\Drafts \HasNoChildren) "/" "[Gmail]/Borradores"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\Flagged \HasNoChildren) "/" "[Gmail]/Destacados"
[DEBUG][2018-02-06T12:22:15.845Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren \Sent) "/" "[Gmail]/Enviados"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren \Important) "/" "[Gmail]/Importantes"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren \Trash) "/" "[Gmail]/Papelera"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren \Junk) "/" "[Gmail]/Spam"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\All \HasNoChildren) "/" "[Gmail]/Todos"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename6"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename7"
[DEBUG][2018-02-06T12:22:15.846Z][1][-redacted-][imap.gmail.com] S: * LSUB (\HasNoChildren) "/" "somename8"
[DEBUG][2018-02-06T12:22:15.847Z][1][-redacted-][imap.gmail.com] S: W6 OK Success
{ root: true, children: [] }

```

**Comment by MarcoCanalcar** [comment]
I opened the #169 PR to fix this!

**Comment by felixhammerl** [comment]
Released: v3.0.5

**Comment by felixhammerl** [comment]
Thanks for the PR 🎉 👍 💯
